src/Basque/Mews.Fiscalizations.Basque/Dto/Bizkaia/Header/BizkaiaHeaderData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Mews.Fiscalizations.Basque.Dto.Bizkaia.Header;
+
+public class BizkaiaHeaderData
+{
+    [JsonPropertyName("con")]
+    public string With { get; set; }
+    
+    [JsonPropertyName("apa")]
+    public string Section { get; set; }
+
+    [JsonPropertyName("inte")]
+    public Inte Inte { get; set; }
+
+    [JsonPropertyName("drs")]
+    public Drs Drs { get; set; }
+}
+
+public class Drs
+{
+    [JsonPropertyName("mode")]
+    public string Mode { get; set; }
```

Comment:
Isn't this the model type 140/240? if yes, then I'd go with an enum or if we're supporting only one of them, then just hardcode it